lib/taskana-core/src/main/java/pro/taskana/common/internal/TaskanaEngineImpl.java

```diff
@@ -348,9 +350,18 @@ protected SqlSessionManager createSqlSessionManager() {
     }
 
     // register type handlers
+    if (DB.isOracleDb(databaseProductName)) {
+      // Use NULL instead of OTHER when jdbcType is not specified for null values,
+      // otherwise oracle driver will chunck on null values
+      configuration.setJdbcTypeForNull(JdbcType.NULL);
+    }
+
     configuration.getTypeHandlerRegistry().register(new MapTypeHandler());
     configuration.getTypeHandlerRegistry().register(Instant.class, new InstantTypeHandler());
     configuration.getTypeHandlerRegistry().register(JdbcType.TIMESTAMP, new InstantTypeHandler());
+    if (DB.isOracle(configuration.getDatabaseId())) {
+      configuration.getTypeHandlerRegistry().register(String.class, new StringTypeHandler());
+    }
```

Comment:
you did this in the HistoryEngineImpl: Merge with the other if statement. For the sake of consistency: do this here to?

Reply:
got it